script/extract-json.mjs

```diff
@@ -0,0 +1,255 @@
+//   }
+// }
+// // console.log(localization);
+
+
+for (const file of jsonFiles) {
+  const filePath = join(process.cwd(), repoPath, 'json', file);
+  try {
+    const { default: jsonData } = await import(filePath, { with: { type: 'json' } });
+    console.log(`Imported ${file}:` /*, jsonData*/);
+
+    defaultRegion[jsonData.language] = jsonData.defaultRegion;
+
+    const voices = jsonData.voices;
+
+    for (const voice of voices) {
+
+      recommended.push({
+        label: voice.label,
+        name: voice.name || undefined,
+        altNames: voice.altNames || undefined,
+        language: voice.language || undefined,
+        gender: voice.gender || undefined,
+        age: voice.age || undefined,
+        quality: Array.isArray(voice.quality) ? voice.quality : [],
+        recommendedPitch: voice.pitchControl === false ? undefined : voice.pitch || 1,
+        recommendedRate: voice.pitchControl === false ? undefined : voice.rate || 1,
+        localizedName: voice.localizedName || "",
+      });
+    }
+
+  } catch (error) {
+    console.error(`Failed to import ${file}: ${error.message}`);
+  }
+}
+
+for (const file of filters) {
+  const filePath = join(process.cwd(), repoPath, 'json', 'filters', file);
+  try {
+    const { default: jsonData } = await import(filePath, { with: { type: 'json' } });
+    console.log(`Imported filters/${file}:` /*, jsonData*/);
+
+    if (file.startsWith("novelty")) {
+      novelty = jsonData.voices.map(({ name, altNames }) => [name, ...(Array.isArray(altNames) ? altNames : [])]).flat();
+    }
+
+    if (file.startsWith("veryLow")) {
+      veryLowQuality = jsonData.voices.map(({ name, language, otherLanguages }) => {
+        // const languages = filterBCP47([language, otherLanguages].flat());
+        // const altNamesGenerated = getAltName(languages);
+        // const altNames = altNamesGenerated.map((v) => name + " " + v);
+        
+        // return [name, altNames].flat();
+        return name;
+      }).flat();
+    }
+  } catch (error) {
+    console.error(`Failed to import filters/${file}: ${error.message}`);
+  }
+}
+
+
+
+const content = `
+
+export const novelty = ${JSON.stringify(novelty)};
+
+export const veryLowQuality = ${JSON.stringify(veryLowQuality)};
+
+export type TGender = "female" | "male" | "nonbinary"
+export type TQuality =  "veryLow" | "low" | "normal" | "high" | "veryHigh";
+
+export interface IRecommended {
+    label: string;
+    name: string;
+    altNames?: string[];
+    language: string;
+    gender?: TGender | undefined;
+    age?: string | undefined;
+    quality: TQuality[];
+    recommendedPitch?: number | undefined;
+    recommendedRate?: number | undefined;
+    localizedName: string;
+};
+      
+export const recommended: Array<IRecommended> = ${JSON.stringify(recommended)};
+
+export const quality = ${JSON.stringify(quality)};
+
+export const defaultRegion = ${JSON.stringify(defaultRegion)};
+`;
+
+const filePath = './src/data.ts';
```

Comment:
This source file is script-generated, may I suggest:
1) adding a comment header at the top of the file that explains which script produced the Typescript code, and at what date/time (or ideally: git revision / commit hash).
2) locate the generated file inside a "gen" subfolder or the "src" source tree, or to rename the file e.g. `src/data.gen.ts`